servers/zts/src/main/java/com/yahoo/athenz/zts/ZTSImpl.java

```diff
@@ -305,8 +308,15 @@ public ZTSImpl(CloudStore implCloudStore, DataStore implDataStore) {
         // make sure to set the keystore for any instance that requires it
         
         setAuthorityKeyStore();
+
+        setNotificationManager();
     }
-    
+
+    private void setNotificationManager() {
```

Comment:
Initiated NotificationManager in ZTS. The daily thread will run but at this point won't send any notifications as the current NotificationTask is not implemented (returns empty collection).